src/api/ledger/accountinfo.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const utils = require('./utils');
+const validate = utils.common.validate;
+const composeAsync = utils.common.composeAsync;
+
+function formatAccountInfo(response) {
+  const data = response.account_data;
+  return {
+    sequence: data.Sequence,
+    xrpBalance: utils.common.dropsToXrp(data.Balance),
+    ownerCount: data.OwnerCount,
+    previousTransactionID: data.PreviousTxnID,
+    previousTransactionLedgerVersion: data.PreviousTxnLgrSeq
```

Comment:
Oh, it wasn't listed in the docs for the account root (https://wiki.ripple.com/Ledger_Format#AccountRoot) so I forgot about it. How is it different than PreviousTxnID? It sounds like it's the same. Does that mean if the asfAccountTxnID wasn't set, then the PreviousTxnID will be undefined?